server/serverRun.py

```diff
@@ -220,6 +220,21 @@ def items():
     return json.dumps(user_items), 200, JSON_CT
 
 
+@app.route('/item/<_id>', methods=['GET'])
```

Comment:
```suggestion
@app.route('/item/<int:_id>', methods=['GET'])
```
This will cast the id to int, (and ensure it is an int)